packages/webpack-plugin/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Codecov
+Copyright (c) 2024 Codecov
```

Comment:
you actually want to use a date range here (`2023-2024`) if you update to latest year.

Reply:
Ouu ty ty for the catch